elements/pl-prairiedraw-figure/pl-prairiedraw-figure.py

```diff
@@ -0,0 +1,45 @@
+import prairielearn as pl
+import lxml.html
+import chevron
+import os
+
+
+def prepare(element_html, element_index, data):
+    element = lxml.html.fragment_fromstring(element_html)
+    required_attribs = ['script_name']
+    optional_attribs = ['params_names', 'width', 'height']
+    pl.check_attribs(element, required_attribs, optional_attribs)
+    return data
+
+
+def render(element_html, element_index, data):
+    element = lxml.html.fragment_fromstring(element_html)
+    script_name = pl.get_string_attrib(element, 'script-name', None)
+    if script_name is None:
```

Comment:
This shouldn't be necessary, we'd never get to the `render` phase if this was missing since it would fail in the `prepare` check.

Reply:
I'll remove the redundant check from `render`.